packages/driver/cypress/fixtures/isolated-runner-inner.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Isolated Runner Fixture</title>
+  </head>
+  <body>
+    Some generic content
+
+    <div>
+      <span class="foo">Foo</span>
+      <span class="bar">Bar</span>
+      <span class="baz">Baz</span>
+      <span class="qux">Qux</span>
```

Comment:
nit: change foo class names etc

Reply:
These actually don't even need to exist in this file.